BSP/STM32F413/Src/BSP_SPI.c

```diff
@@ -126,3 +135,34 @@ void BSP_SPI_SetStateCS(uint8_t state) {
         GPIO_ResetBits(GPIOB, GPIO_Pin_6);
     }
 }
+
+/***************THE FOLLWING CODE IS FOR THE RTOS VERSION OF THE BPS*********/
+#ifdef RTOS
+
+void SPI1_IRQHandler(void){
```

Comment:
I'm not sure if this was on purpose, but if the ifdef RTOS is here, then we would not have an IRQ for SPI when using baremetal code. If we do not want an IRQ for SPI with bare metal than this will work